core/net/ipv6/uip-ds6-route.c

```diff
@@ -297,11 +303,16 @@ uip_ds6_route_add(uip_ipaddr_t *ipaddr, uint8_t length,
        least recently used one we have. */
 
     if(uip_ds6_route_num_routes() == UIP_DS6_ROUTE_NB) {
+      uip_ds6_route_t *oldest;
+      oldest = NULL;
+#if UIP_DS6_ROUTE_REMOVE_LEAST_RECENTLY_USED
```

Comment:
I'd be ok with the whole code logic removed. It probably never is a good idea to have this module delete old routes but rather leave all responsibility to the routing protocol. I'd also add a comment